src/NzbDrone.Core/MediaFiles/UpdateEpisodeFileService.cs

```diff
@@ -22,11 +22,12 @@ public interface IUpdateEpisodeFileService
     public class UpdateEpisodeFileService : IUpdateEpisodeFileService,
                                             IHandle<SeriesScannedEvent>
     {
+        internal static readonly DateTime EpochTime = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
```

Comment:
This should be moved else where, instead of being made internal if it's being used for tests.